devel/ldid-procursus/Portfile

```diff
@@ -7,9 +7,10 @@ PortGroup           makefile 1.0
 
 name                ldid-procursus
 version             2.1.5-procursus7
-revision            1
+revision            2
```

Comment:
why are you increasing the `revision` here - there shouldn't be a need for that. I didn't build before on the earlier OSes and will not now either.